js/renovation/ui/common/__tests__/widget.test.tsx

```diff
@@ -544,6 +546,45 @@ describe('Widget', () => {
         });
       });
 
+      describe('windowResizeEffect', () => {
+        afterEach(() => {
+          jest.clearAllMocks();
+        });
+
+        it('should subscribe to windowResizeEffect event', () => {
+          const onDimensionChanged = jest.fn();
+          const widget = new Widget({ onDimensionChanged });
+
+          const dispose = widget.windowResizeEffect() as DisposeEffectReturn;
+
+          expect(resizeCallbacks.add).toBeCalledTimes(1);
+          expect(resizeCallbacks.remove).toBeCalledTimes(0);
+
+          dispose?.();
+        });
+
+        it('should return unsubscribe callback', () => {
```

Comment:
`should return window.onresize unsubscribe callback`

Reply:
https://github.com/DevExpress/DevExtreme/pull/16330/commits/fcb3408ec98d8bb1cbf5cac06409c49357272be7